src/protagonist/DLCS.Model/Spaces/ISpaceRepository.cs

```diff
@@ -12,6 +12,23 @@ public interface ISpaceRepository
     
     Task<Space?> GetSpace(int customerId, int spaceId, bool noCache, CancellationToken cancellationToken);
     
+    /// <summary>
+    /// Retrieves a space by name
+    /// </summary>
+    /// <param name="customerId">The customer to retrieve a space from</param>
+    /// <param name="name">The name of the space</param>
+    /// <param name="noCache">Whether the value should be cached</param>
+    /// <param name="cancellationToken">The cancellation token</param>
+    /// <returns>A space, or null if it can't be found</returns>
+    Task<Space?> GetSpace(int customerId, string name,  bool noCache, CancellationToken cancellationToken);
```

Comment:
Is it worth introducing an overload that isn't used? Would it be better to introduce it when/if it's required?

Reply:
I added this in order to mirror the `id` methods provided, but thinking about it now, it's not being used so probably better to remove it